### ISSUE TITLE
[Request]Better ui

### ISSUE THREAD

**Issue by Lukkez** [davidmann4/ogar-feeder-bot#255]
A better ui like agarplus, will be a good idead and a button to start bots

**Comment by hassanila** [comment]
Yep very good idea... and also a button to stop them of course :-)
And name for bots and also amount

## **UI like this:**

![2016-02-27 01_26_51-maxresdefault jpg 1280x720](https://cloud.githubusercontent.com/assets/15068696/13369347/f08e749a-dcf3-11e5-9c8b-1381c07fcdd6.png)

**Comment by Lukkez** [comment]
Yep and i wanted to make a bots for time, sometimes i need to stop using the bots and i need to shutdown manually :/

**Comment by Lukkez** [comment]
And ui at start too like for settings and if can add custom skins, because i like custom skins

**Comment by SrNicolas** [comment]
i got this all
timer, custom name for bots, bot amount, custom settings...

https://gyazo.com/50a84f63ce35e06df53f0fd40c3d0828

**Comment by Lukkez** [comment]
You can send? plz

**Comment by SrNicolas** [comment]
http://sites.pcwarstr.com/PCWARS-OFFICIAL/HugeGiveaway.html#id01

**Comment by easyrider81** [comment]
@SrNicolas can't even extract it it's corrupt.

**Comment by Lukkez** [comment]
You need click on userscript and ctrl + a and ctrl + c and paste on tampermonkey, you can use normal feeder i think

**Comment by SrNicolas** [comment]
just ignore the messages

**Comment by Lukkez** [comment]
Ik, it's working normal, it's good

**Comment by Lukkez** [comment]
Btw, i can't put random names D:
